src/content/docs/reference/Uniforms/General/heldBlockLightValue.mdx

```diff
@@ -4,9 +4,7 @@ description: heldBlockLightValue
 sidebar:
   label: heldBlockLightValue
   order: 1
-  badge:
-    text: Unfinished
-    variant: caution
 ---
 
-### `uniform int heldBlockLightValue;`
\ No newline at end of file
+### `uniform int heldBlockLightValue;`
+The light strength of the item held in the player's hand.
```

Comment:
This should indicate the valid range of the variable, in this case 0 to 15 for vanilla blocks and potentially more for modded blocks